pkgs/stdenv/generic/setup.sh

```diff
@@ -1305,6 +1305,26 @@ showPhaseHeader() {
 }
 
 
+showPhaseFooter() {
+    function displaytime {
```

Comment:
shell functions are not lexically scoped, since this is used only once it might be better to inline the single use instead?

Reply:
done